replica/table.cc

```diff
@@ -526,8 +527,9 @@ std::vector<std::unique_ptr<compaction_group>> table::make_compaction_groups() {
     std::vector<std::unique_ptr<compaction_group>> ret;
     auto&& ranges = dht::split_token_range_msb(_x_log2_compaction_groups);
     tlogger.debug("Created {} compaction groups for {}.{}", ranges.size(), _schema->ks_name(), _schema->cf_name());
+    auto gid = compaction::group_id(ranges.size());
```

Comment:
Okay, I see why you added that constructor. But it's not intuitive. Better to initialize it from 0, size.

Reply:
Or even, instead of a funky type just have a string there. Then we can later put the table name + tablet id, or whatever. Only this emplace_back needs to worry about it.

Reply:
okay, sounds good